<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QSlider::groove:horizontal{
background:#000000;
height:8px;
border-radius:4px;
}

QSlider::add-page:horizontal{
background:#E4E4E4;
height:8px;
border-radius:4px;
}

QSlider::sub-page:horizontal{
background:#000000;
height:8px;
border-radius:4px;
}

QSlider::handle:horizontal{
width:13px;
margin-top:-3px;
margin-bottom:-3px;
border-radius:6px;
background:qradialgradient(spread:pad,cx:0.5,cy:0.5,radius:0.5,fx:0.5,fy:0.5,stop:0.6 #FFFFFF,stop:0.8 #B6B6B6);
}
QPushButton {
background-color:#ffffff;
border-width:2px;
border-color: #000000;
border-style: solid;
border-radius:10px;  
}
QGroupBox{
    border:2px solid;
    border-radius:5px;
    margin-top:0.5em;
}

QGroupBox::title{
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding:0 1px;
}
QSlider::groove:horizontal{
background:#000000;
height:8px;
border-radius:4px;
}

QSlider::add-page:horizontal{
background:#E4E4E4;
height:8px;
border-radius:4px;
}

QSlider::sub-page:horizontal{
background:#000000;
height:8px;
border-radius:4px;
}

QSlider::handle:horizontal{
width:13px;
margin-top:-3px;
margin-bottom:-3px;
border-radius:6px;
background:qradialgradient(spread:pad,cx:0.5,cy:0.5,radius:0.5,fx:0.5,fy:0.5,stop:0.6 #FFFFFF,stop:0.8 #B6B6B6);
}
QPushButton {
background-color:#ffffff;
border-width:2px;
border-color: #000000;
border-style: solid;
border-radius:10px;  
}
background-image: url(:/images/bg.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDockWidget" name="dockWidget_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>172</width>
      <height>621</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>172</width>
      <height>46</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
    </property>
    <property name="windowTitle">
     <string> 工具箱</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_2">
     <widget class="QPushButton" name="pushButton_origin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>显示原图</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cam">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>打开摄像头</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_choose">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color:#ffffff;
border-width:2px;
border-color: #000000;
border-style: solid;
border-radius:10px;  
}</string>
      </property>
      <property name="text">
       <string>选择图片</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_fliter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>滤波</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_noi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>噪声</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cut">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>裁剪</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_concat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>图片拼接</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sumiao">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>素描效果</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_edge">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>边框</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_face">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>人脸美化</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_mirrorY">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>490</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:32px;
background-repeat:no-repeat;
background-position:center;
background-image: url(:/images/chuizhi.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_turnright">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>410</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:32px;
background-repeat:no-repeat;
background-position:center;
background-image: url(:/images/turn-right.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_turnleft">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:32px;
background-repeat:no-repeat;
background-position:center;
background-image: url(:/images/turn-left.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_mirrorX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:32px;
background-repeat:no-repeat;
background-position:center;
background-image: url(:/images/shuiping.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_show">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>791</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDockWidget" name="dockWidget_2">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>0</y>
      <width>172</width>
      <height>621</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>172</width>
      <height>46</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
    </property>
    <property name="windowTitle">
     <string> 工具箱</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_3">
     <widget class="QSlider" name="horizontalSlider_halation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>晕影</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_contour">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>描边阈值</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_saturation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_satucomfirm">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>220</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:32px;
background-repeat:no-repeat;
background-position:center;
background-image: url(:/images/);</string>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>对比度</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_contrast">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>饱和度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>亮度</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_lightcomfirm">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:32px;
background-repeat:no-repeat;
background-position:center;
</string>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_light">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>110</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal{
background:#000000;
height:8px;
border-radius:4px;
}

QSlider::add-page:horizontal{
background:#E4E4E4;
height:8px;
border-radius:4px;
}

QSlider::sub-page:horizontal{
background:#000000;
height:8px;
border-radius:4px;
}

QSlider::handle:horizontal{
width:13px;
margin-top:-3px;
margin-bottom:-3px;
border-radius:6px;
background:qradialgradient(spread:pad,cx:0.5,cy:0.5,radius:0.5,fx:0.5,fy:0.5,stop:0.6 #FFFFFF,stop:0.8 #B6B6B6);
}
</string>
      </property>
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_contrastcomfirm">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:32px;
background-repeat:no-repeat;
background-position:center;
background-image: url(:/images/);</string>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_satucomfirm_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>280</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:32px;
background-repeat:no-repeat;
background-position:center;
background-image: url(:/images/);</string>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_satucomfirm_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>340</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:32px;
background-repeat:no-repeat;
background-position:center;
background-image: url(:/images/);</string>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_cam_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>570</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:#ffffff;
border-width:2px;
border-color: #000000;
border-style: solid;
border-radius:10px;  
}</string>
    </property>
    <property name="text">
     <string>保存图片</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1192</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopenfile"/>
    <addaction name="actionsavefile"/>
    <addaction name="actioncamera"/>
    <addaction name="actiondock_show"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionopenfile">
   <property name="text">
    <string>打开图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionsavefile">
   <property name="text">
    <string>保存图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actiondock_show">
   <property name="text">
    <string>显示工具箱</string>
   </property>
   <property name="toolTip">
    <string>dock_show</string>
   </property>
  </action>
  <action name="actioncamera">
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
